docs/changes/v11.9.0/Tools-Changes.adoc

```diff
@@ -0,0 +1,18 @@
+= Tools Changes =
+
+== Update subsystem ACL ==
+
+The following commands have been added to find, add and delete ACL:
+
+- `pki-server <subsystem>-acl-find`;
+- `pki-server <subsystem>-acl-add`;
+- `pki-server <subsystem>-acl-delete`.
```

Comment:
@fmarco76 I'm trying to find the matching commands just to be sure.  I didn't find any reference of "acl-delete" in the src but found "acl-del". Is it misspelled in the doc or are they made equal at some other code location?